tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -614,8 +615,8 @@ async def test_doesnt_checkpoint_with_zero_eval_num_examples(
 @pytest.mark.parametrize(
     "classifier_params",
     [
-        {RANDOM_SEED: 1, EPOCHS: 1, BILOU_FLAG: False},
-        {RANDOM_SEED: 1, EPOCHS: 1, BILOU_FLAG: True},
+        {RANDOM_SEED: 1, EPOCHS: 1, BILOU_FLAG: False, NUM_TRANSFORMER_LAYERS: 0},
```

Comment:
are the layers tested somewhere else? (or do we not need to test them)

Reply:
There are 10+ tests that train DIET and none of them has transformer layers disabled. There's actually probably some potential to create some unit testing config with reduced parameter counts that is reused across these tests. 

I just looked at the dashboard and saw that this test was very slow and assumed that the transformer layer is probably the slowest part here right now and wanted to see what happens.